systemtest/070-list-tags.bats

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bats
+#
+# list-tags tests
+#
+
+load helpers
+
+function setup() {
+    standard_setup
+}
+
+# list from registry
+@test "list-tags: remote repository on a registry" {
+    local remote_image=docker://quay.io/libpod/alpine_labels
+    local output=
+
+    run_skopeo list-tags --tls-verify=false $remote_image
+    expect_output --substring "quay.io/libpod/alpine_labels"
+    expect_output --substring "latest"
+}
+
+# list from a local docker-archive file
+@test "list-tags: from a docker-archive file" {
+    local file_name=./systemtest/testdata/docker-two-images.tar.xz
```

Comment:
Tests are not guaranteed to run while cd'ed to this directory. You must use an absolute path for this:
```bash
    local file_name=${TEST_SOURCE_DIR}/testdata/docker-two-images.tar.xz
```
And actually, since this string is only used in one place, there isn't really a need for it to be a variable.

Reply:
> Tests are not guaranteed to run while cd'ed to this directory. You must use an absolute path for this:
> 
> ```shell
>     local file_name=${TEST_SOURCE_DIR}/testdata/docker-two-images.tar.xz
> ```
> 
> And actually, since this string is only used in one place, there isn't really a need for it to be a variable.

done